website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 
+
+- [`spec.requirements`](../reference/provisioner-crd#specrequirements)
+  - Cloud Provider Agnostic 
+  - Kubernetes Well Known Labels
+  - This section includes generally applicable constraints (zone, instance type) that each cloud provider is expected to implement. 
+  - Reference the Provisioner CRD for more information. 
+- [`spec.provider`](#specprovider)
+  - Cloud Provider Specific
+  - This section defines constraints that are unique to AWS, such as SecurityGroups.
+  - Defined below. 
+
+## spec.provider

```

Comment:
I'd shortcircuit everything above and just talk about `provider` and provider specific things.

> Karpenter uses Kubernetes' `runtime.RawExtensions` to extend the [Provisioner CRD](../reference/provisioner-crd#s) with vendor specific configuration in [spec.provider].(https://github.com/awslabs/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L33). This section covers parameters of the AWS Cloud Provider.

Reply:
I agree that we should cut the bullet list and just start in on the spec.provider content.